test/images/agnhost/netexec/netexec.go

```diff
@@ -731,6 +743,14 @@ func getHostName() string {
 	return hostName
 }
 
+func getNodeName() string {
+	nodeName := os.Getenv("NODE_NAME")
+	if nodeName == "" {
+		log.Fatalf("env var NODE_NAME was empty; must be set to use nodename endpoints")
+	}
+	return nodeName
```

Comment:
this is problematic for an e2e test to depend on the actual manifest of the pod and panic if the env does not exist